tests/functional/disaster-recovery/regional-dr/test_warning_and_alerting.py

```diff
@@ -348,3 +349,202 @@ def test_rdr_inconsistent_data_warning_alert(
+            num_of_subscription=1,
+            num_of_appset=0,
+            pvc_interface=constants.CEPHBLOCKPOOL,
+        )
+        workload_names.append(f"{rdr_workload[0].workload_name}-1")
+        dr_workload(
+            num_of_subscription=0,
+            num_of_appset=1,
+            pvc_interface=constants.CEPHFILESYSTEM,
+        )
+        workload_names.append(f"{rdr_workload[1].workload_name}-1-cephfs")
+
+        logger.info(f"Workload names are {workload_names}")
+
+        dr_helpers.set_current_primary_cluster_context(
+            rdr_workload[0].workload_namespace
+        )
+
+        scheduling_interval = dr_helpers.get_scheduling_interval(
+            rdr_workload[0].workload_namespace
+        )
+        wait_time = 2 * scheduling_interval  # Time in minutes
+        logger.info(f"Waiting for {wait_time} minutes to run IOs")
+        sleep(wait_time * 60)
+
+        primary_cluster_name = dr_helpers.get_current_primary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        secondary_cluster_name = dr_helpers.get_current_secondary_cluster_name(
+            rdr_workload[0].workload_namespace, rdr_workload[0].workload_type
+        )
+
+        config.switch_acm_ctx()
+        acm_obj = AcmAddClusters()
+        page_nav = ValidationUI()
+
+        ocp_version = get_ocp_version()
+        acm_loc = locators[ocp_version]["acm_page"]
+
+        page_nav.refresh_web_console()
+        config.switch_to_cluster_by_name(primary_cluster_name)
+        drpc_subscription = DRPC(namespace=rdr_workload[0].workload_namespace)
+        drpc_appset = DRPC(
+            namespace=constants.GITOPS_CLUSTER_NAMESPACE,
+            resource_name=f"{rdr_workload[1].appset_placement_name}-drpc",
+        )
+        drpc_objs = [drpc_subscription, drpc_appset]
+        before_failover_last_group_sync_time = []
+        for obj in drpc_objs:
+            before_failover_last_group_sync_time.append(
+                dr_helpers.verify_last_group_sync_time(obj, scheduling_interval)
+            )
+        logger.info("Verified lastGroupSyncTime")
+        logger.info(
+            "Scale down rbd-mirror deployment on the secondary cluster and mds deployments on the primary cluster"
+        )
+        config.switch_to_cluster_by_name(secondary_cluster_name)
+        helpers.modify_deployment_replica_count(
+            deployment_name=constants.RBD_MIRROR_DAEMON_DEPLOYMENT, replica_count=0
+        )
+        config.switch_to_cluster_by_name(primary_cluster_name)
+        helpers.modify_deployment_replica_count(
+            deployment_name=constants.MDS_DAEMON_DEPLOYMENT_ONE, replica_count=0
+        )
+        helpers.modify_deployment_replica_count(
+            deployment_name=constants.MDS_DAEMON_DEPLOYMENT_TWO, replica_count=0
+        )
+        config.switch_acm_ctx()
+        logger.info(f"Waiting for {wait_time * 60} seconds to allow alert to appear")
+        sleep(wait_time * 60)
```

Comment:
Here, we are waiting for 2x of the scheduled interval but in the description it has been mentioned 3x.